### PR TITLE
Adsk Contrib - Improve the Matrix value precision in config file

### PR DESCRIPTION
**Pull request by hodoulp**
Improve the precision of the Matrix values in a config file to overcome the 6-digits limitation from YAML current version.  As the underlying MatrixOp was already moved to double to increase internal computation (i.e. combine, ...) precision, the MatrixTransform now provides double precision access.

*Note:* That's a better long-term solution compare to 'patch' the current YAML library as it was previously discussed.  With the current implementation, the precision is now preserved whatever is the used YAML version.

This will also address Issue #577.

### CODE REVIEW

**Comment by doug-walker** [comment]
@KevinJW, this is our preferred solution to solving this issue.  Since you wrote PR #389, could you please review?

File: src/OpenColorIO/MathUtils.h [hunk 1]
```diff
@@ -30,101 +30,89 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
...
 #define INCLUDED_OCIO_MATHUTILS_H
 
-#include <cmath>
-#include <vector>
+#if defined(_WIN32) || defined(_WIN64) || defined(_WINDOWS) || defined(_MSC_VER)
+#ifndef WINDOWS
+#define WINDOWS
+#endif
+#endif
+
```
Comment (KevinJW) [inline src/OpenColorIO/MathUtils.h]: Is this to avoid min/max pollution or something?  would defining  NOMINMAX be less ugly if so?

Comment (hodoulp) [reply src/OpenColorIO/MathUtils.h]: That's dead code I forgot to remove in my various attempts to fix the Clang 9 warnings.

Comment (hodoulp) [reply src/OpenColorIO/MathUtils.h]: Fixed.

**Comment by hodoulp** [comment]
@KevinJW I did the change, and plan to merge it next tuesday. Please let me know if you have any comment.

**Comment by hodoulp** [comment]
Following the two-weeks rule, here is a friendly reminder that the branch could be merged this week. Unless there are some objections I will merge it tomorrow afternoon (New-York timezone).

**Pull request merged**